src/VscPreviewExtension/package.json

```diff
@@ -2,23 +2,36 @@
   "name": "DfmPreview",
   "displayName": "DfmPreview",
   "description": "Preview the DocFX Flavored Markdown and token tree in Visual Studio Code",
-  "version": "0.0.21",
-  "publisher": "928pjy",
+  "version": "0.0.1",
```

Comment:
it's ok for me

---
In reply to: [82717952](https://github.com/dotnet/docfx/pull/763#discussion_r82717952) [](ancestors = 82717952)